src/Lecture1.hs

```diff
@@ -50,7 +51,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Num a => a -> a -> a
```

Comment:
`Num` is explained in the following lectures, so you are one step ahead 😸
In here `Int -> Int -> Int` or `Integer -> Integer -> Integer` would also work, as a more specific type 🙂